content/2021-online/submit.md

```diff
@@ -0,0 +1,24 @@
+---
+title: Submit Talk
+---
+
+## Submit a talk
+
+The PromCon Call for Papers (CfP) is open for talk submissions until 23:59 UTC on
+Friday, April 4, 2021, and we encourage you to submit a talk!
+
+<a class="btn btn-default btn-lg" href="https://docs.google.com/forms/d/1mKYcdSIw02Dq_uP2gKOiHoOnOQr4W5jwXqdT_Zsk41Y/viewform?edit_requested=true" target="_blank" role="button">
```

Comment:
Is that `?edit_requested=true` intentional at the end?

Reply:
Nope, removed